boltzrpc/boltzrpc.proto

```diff
@@ -409,6 +415,15 @@ message ListSwapsResponse {
     repeated ChainSwapInfo chain_swaps = 4;
 }
 
+message GetStatsRequest {
+    // only consider manual or autoswaps. returns both if left empty
+    optional bool is_auto = 1;
```

Comment:
I'd prefer an enum. Optional values in gRPC are weird in some languages

Reply:
I think its fine, we have the same in `ListSwaps` and optional values in general all over the place. How would you name the enum?

Reply:
- `ALL`
- `AutoSwap`
- `Manual`